nextflow_schema.json

```diff
@@ -1058,5 +1070,11 @@
         {
             "$ref": "#/definitions/generic_options"
         }
-    ]
+    ],
+    "properties": {
+        "validationSchemaIgnoreParams": {
+            "type": "string",
+            "default": "cf_ploidy,genomes,igenomes_base"
+        }
+    }
```

Comment:
where is this coming from?

Reply:
I have absolutely no Idea, I will remove